src/codewhisperer/activation.ts

```diff
@@ -202,10 +202,7 @@ export async function activate(context: ExtContext): Promise<void> {
         )
     )
 
-    if (!isCloud9() && !auth.isConnectionValid()) {
-        // this is to proactively check and reflect the state if user's connection is expired
-        auth.refreshCodeWhisperer()
-    }
+    await auth.restore()
```

Comment:
do we need a Cloud9 check here?

Reply:
I believe this has to be tested in C9 to ensure it is not breaking.